apollo-router/src/plugins/telemetry/dynamic_attribute.rs

```diff
@@ -240,7 +240,9 @@ impl EventDynAttribute for ::tracing::Span {
         self.with_subscriber(move |(id, dispatch)| {
             if let Some(reg) = dispatch.downcast_ref::<Registry>() {
                 match reg.span(id) {
-                    None => eprintln!("no spanref, this is a bug"),
+                    None => {
+                        eprintln!("no spanref, this is a bug");
```

Comment:
This should panic in debug builds?

Reply:
I'll do this in a follow up or as it's not a change